[2023-08-31 13:21:55,119] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-07T00:00:00+00:00 [queued]>
[2023-08-31 13:21:55,199] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2021-09-07T00:00:00+00:00 [queued]>
[2023-08-31 13:21:55,201] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:21:55,201] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-08-31 13:21:55,202] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-08-31 13:21:55,283] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-07T00:00:00+00:00
[2023-08-31 13:21:55,292] {standard_task_runner.py:52} INFO - Started process 2343 to run task
[2023-08-31 13:21:55,311] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2021-09-07T00:00:00+00:00', '--job-id', '121', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpdonqf1br', '--error-file', '/tmp/tmp5fjihm7r']
[2023-08-31 13:21:55,325] {standard_task_runner.py:77} INFO - Job 121: Subtask second_task
[2023-08-31 13:21:55,507] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2021-09-07T00:00:00+00:00 [running]> on host e7718740c1e6
[2023-08-31 13:21:55,709] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=devendra
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-07T00:00:00+00:00
[2023-08-31 13:21:55,713] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-08-31 13:21:55,714] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2023-08-31 13:21:55,735] {bash.py:169} INFO - Output:
[2023-08-31 13:21:55,850] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2023-08-31 13:21:55,868] {bash.py:177} INFO - Command exited with return code 0
[2023-08-31 13:21:55,990] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20210907T000000, start_date=20230831T132155, end_date=20230831T132155
[2023-08-31 13:21:56,555] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-31 13:21:56,600] {local_task_job.py:146} INFO - Task exited with return code 0
